pkg/suggestion/v1beta1/chocolate/base_service.py

```diff
@@ -157,9 +157,23 @@ def getSuggestions(self, trials, request_number):
                     logger.info("New record in sqlite DB is updated")
                     logger.info("{}\n".format(
                         self.created_trials[new_trial_loss_idx]))
+        # Assuming that created_trials are already populated
+        # TODO: Handle Restart of algorithm pod
+        logger.info("{} Trials created in DB".format(len(self.created_trials)))
+        new_actual_requested_no = total_request_number - len(self.created_trials)
+        prev_generated_no = request_number - new_actual_requested_no
+        logger.info("In this call, New {} Trials will be generated, {} Trials will be reused from previously generated".format(new_actual_requested_no, prev_generated_no))
```

Comment:
In the first call, `new_actual_requested_no` = 0 and `prev_generated_no` = 3, when `request_number` = 3.
Is that correct ?

Reply:
It is the other way around. 
In the normal case,  
         total_request_number == len(self.created_trials) + request_number where self.created_trials correspond to the number of previously created trials.  So, prev_generated_no will be 0 in this case 

When there is a difference, it means that some of the suggestions in self.created_trials(same as in DB)  are not recorded in K8s Suggestions resource. So,  prev_generated_no will be greater than 0 in this case